werkzeug/http.py

```diff
@@ -988,6 +988,9 @@ def dump_cookie(key, value='', max_age=None, expires=None, path='/',
     elif max_age is not None and sync_expires:
         expires = to_bytes(cookie_date(time() + max_age))
 
+    if samesite:
+        samesite = 'Strict'
```

Comment:
This is broken. You have an option that appears to take a boolean or a string but if it's truthy you always set it to strict. I'd go for this:

```
if samesite not in (None, 'Strict', 'Lax'):
    raise ValueError("invalid samesite value; must be 'Strict', 'Lax' or None")
```

or this:

```
if samesite and samesite not in ('Strict', 'Lax'):
    raise ValueError("invalid samesite value; must be 'Strict', 'Lax' or None")
```